src/components/Button.jsx

```diff
@@ -0,0 +1,13 @@
+function Button({ clickHandler, children, type, disabled }) {
+    return (
+        <button
+            onClick={clickHandler}
+            type={type}
+            disabled={disabled || false}
+        >
+            {children}
+        </button>
+    );
+}
```

Comment:
Dit is nu een soort doorgeefluik. Je had net zo goed direct het button element kunnen gebruiken